<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1736050944823' stop='1736050945217'>
    <name>Logout functionality</name>
    <title>Logout functionality</title>
    <test-cases>
        <test-case start='1736050944853' status='broken' stop='1736050945212'>
            <name>Display About page</name>
            <title>Display About page</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>`cy.visit()` failed trying to load:
                
                /
                
                We failed looking for this file at the path:
                
                /home/diniranga/Desktop/Group23_Cypress_Cucumber-dev/Cypress_Cucumber_Test/
                
                The internal Cypress web server responded with:
                
                  > 404: Not Found</message>
                <stack-trace>CypressError: `cy.visit()` failed trying to load:
                
                /
                
                We failed looking for this file at the path:
                
                /home/diniranga/Desktop/Group23_Cypress_Cucumber-dev/Cypress_Cucumber_Test/
                
                The internal Cypress web server responded with:
                
                  > 404: Not Found
                    at &lt;unknown> (http://localhost:45969/__cypress/runner/cypress_runner.js:131591:76)
                    at visitFailedByErr (http://localhost:45969/__cypress/runner/cypress_runner.js:131006:12)
                    at &lt;unknown> (http://localhost:45969/__cypress/runner/cypress_runner.js:131574:13)
                    at tryCatcher (http://localhost:45969/__cypress/runner/cypress_runner.js:1830:23)
                    at Promise._settlePromiseFromHandler (http://localhost:45969/__cypress/runner/cypress_runner.js:1542:31)
                    at Promise._settlePromise (http://localhost:45969/__cypress/runner/cypress_runner.js:1599:18)
                    at Promise._settlePromise0 (http://localhost:45969/__cypress/runner/cypress_runner.js:1644:10)
                    at Promise._settlePromises (http://localhost:45969/__cypress/runner/cypress_runner.js:1720:18)
                    at _drainQueueStep (http://localhost:45969/__cypress/runner/cypress_runner.js:2430:12)
                    at _drainQueue (http://localhost:45969/__cypress/runner/cypress_runner.js:2423:9)
                    at Async._drainQueues (http://localhost:45969/__cypress/runner/cypress_runner.js:2439:5)
                    at Async.drainQueues (http://localhost:45969/__cypress/runner/cypress_runner.js:2309:14)
                From Your Spec Code:
                    at Context.eval (cypress/e2e/TestCases/about/about.js:4:7)
                    at Context.resolveAndRunStepDefinition (node_modules/cypress-cucumber-preprocessor/lib/resolveStepDefinition.js:193:0)
                    at Context.eval (node_modules/cypress-cucumber-preprocessor/lib/createTestFromScenario.js:27:0)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>